### ISSUE TITLE
Unable to create xls containing Property definitions

### ISSUE THREAD

**Issue by mdenker** [INM-6/python-odmltables#6]
Trying to convert odml to xls with Property Definitions enabled leads (with files of L/N) to:

```
Traceback (most recent call last):
  File "/home/denker/Projects/toolboxes/py/python-odmltables/odmltables/wizard/converterpages.py", line 1052, in handlebuttonbrowse
    convert(self.settings)
  File "/home/denker/Projects/toolboxes/py/python-odmltables/odmltables/wizard/converterpages.py", line 1288, in convert
    table.write2file(settings.get_object('outputfilename'))
  File "/home/denker/Projects/toolboxes/py/python-odmltables/odmltables/odml_xls_table.py", line 310, in write2file
    sheet.col(i).width = 256 * (l + 1)
  File "/home/denker/anaconda/envs/stdpy2_qt4/lib/python2.7/site-packages/xlwt/Column.py", line 24, in set_width
    raise ValueError("column width (%r) not an int in range(65536)" % width)
ValueError: column width (125696) not an int in range(65536)
```

**Comment by JuliaSprenger** [comment]
This sounds like the odml you are trying to convert is too big in size. Does this also occur for small files?

**Comment by mdenker** [comment]
The problem is that the xls columns were made as long as the largest entry -- this is sometimes too long. I capped it to 80 characters. Hopefully fixed by #7
